package com.zy.junit;

import org.junit.Test;

import java.io.IOException;

/**
 * <ul>
 * <li>文件包名 : com.zy.junit</li>
 * <li>创建时间 : 2019/10/29 9:51</li>
 * <li>修改记录 : 无</li>
 * </ul>
 * 类说明：
 *
 * @author zhengyu
 */
public class RuntimeTest {

    @Test
    public void runtimeTest() throws Exception {
       /* Runtime runtime = Runtime.getRuntime();
        Process process= runtime.exec("cmd /k start dir");
        if(process.waitFor() == 0){
            System.out.println("正常线程结束");
        }else{
            System.out.println(process);
        }
        System.out.println("xxxx");*/
    }
}
